math/calculator.cpp

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright 2020 @author David Leal
+ * @file
+ *
+ * Basic calculator: sum, subtract, multiply, and divide.
+ */
+
+#include <iostream>
+#include <conio.h>
```

Comment:
this library is only present in MSVC and not a part of ISO C++ standards - https://en.wikipedia.org/wiki/Conio.h